shared/desktop/renderer/index.js

```diff
@@ -74,12 +74,12 @@ function setupApp(store) {
 
   ipcRenderer.send('mainWindowWantsMenubarWindowID')
 
-  // After a delay, see if we're connected, and try starting keybase if not
-  setTimeout(() => {
+  // See if we're connected, and try starting keybase if not.
+  setImmediate(() => {
```

Comment:
why do we not need to delay anymore?

Reply:
Before, watchdog used to kill + restart the processes, so the delay was to wait and see if they were already running and becoming connected. Now with the instance check, the second watchdog is harmless if there already was one. Note this only has an effect on Windows.